libraries/testing/src/fakeauth.js

```diff
@@ -38,6 +38,13 @@ exports.start = function(clients, { rootUrl } = {}) {
           scopes = clients[clientId];
           ext = bewitParts[3] || '';
         }
+      } else {
+        return {
+          status: 'no-auth',
+          scheme: 'none',
+          scopes: anonymousScopes || [],
```

Comment:
should this default to `[assume:anonymous]` just to be parallel to the actual implementation?

Reply:
I don't remember why this was done but perhaps it was done to allow calling this function with no scopes at all since some endpoints actually don't require any scopes.

Reply:
I'm thinking we should have it do what auth actually does. (that is if I understand auth's implementation correctly.

Reply:
Changing this to `scopes: ['assume:anonymous']` seems to have broken the tests.